Makefile

```diff
@@ -31,6 +31,6 @@ cli:
 	# build documentation
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	# write out dict of availabe versions and formats
-	bin/write-version-file doc/versions.json
+	bin/write-version-file > doc/versions.json
```

Comment:
Interesting, not related to your change, but I think it was previously broken? I see here we create `versions.json`, with a list of versions and formats.

But in other pages we reference `versions.js`, which does not exist. At least after running `make html singlehtml slides` then serving the content from `cylc-doc/doc/8.0a0/html` with `python -m http.server 8000` I get 404 for `http://localhost:8000/versions.js` in the browser console. Also found no reference of `versions.json` in the `src` directory.

![image](https://user-images.githubusercontent.com/304786/61769149-70c97600-ae3d-11e9-9ba6-a19ca26118a3.png)

Reply:
Aha, looks like the `versions.js` file was excluded by the `.gitignore`, fixed.